include/cantera/thermo/BinarySolutionTabulatedThermo.h

```diff
@@ -147,12 +147,19 @@ class BinarySolutionTabulatedThermo : public IdealSolidSolnPhase
     virtual void getParameters(AnyMap& phaseNode) const;
     virtual void initThermoXML(XML_Node& phaseNode, const std::string& id_);
 
+    void getIntegratedPartialMolarVolumes(doublereal* vbar);
+    void getPartialMolarVolumes(doublereal* vbar) const;
+    virtual void calcDensity();
+
 protected:
     //! If the compositions have changed, update the tabulated thermo lookup
     virtual void compositionChanged();
 
     //! Species thermodynamics interpolation functions
-    std::pair<double,double> interpolate(double x) const;
+    double interpolate(double x,const vector_fp& molefrac,const vector_fp& inputData) const;
```

Comment:
Function definitions should please have one space after a comma. I'm not sure exactly how long this line is, but we prefer to keep line lengths to 88 characters or less, if possible.

Reply:
In other words:

```suggestion
    double interpolate(double x, const vector_fp& molefrac,
        const vector_fp& inputData) const;
```

There is a similar issue on line 162 (next declaration)